src/app.js

```diff
@@ -24,7 +24,7 @@ import { noSearchEngineIndex } from './middleware/noSearchEngineIndex.js'
 import { RateLimitError } from './utils/error.js'
 
 const app = express()
-const isDev = AppConstants.NODE_ENV === 'dev'
+const isDev = AppConstants.NODE_ENV === 'development'
 
 // init sentry
 Sentry.init({
```

Comment:
For sentry, we might have to use `APP_ENV` to differencitate

Reply:
OK I looked in docs, we can pass `environment` to the `Sentry.init` function and use `APP_ENV`.